alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -272,6 +273,9 @@ private function initMembers()
 		$uiConf = uiConfPeer::retrieveByPK($this->uiconfId);
 		if (!$uiConf)
 			KExternalErrors::dieError(KExternalErrors::UI_CONF_NOT_FOUND);
+
+		$playerConfig = json_decode($uiConf->getConfig(), true);
```

Comment:
the uiconf config can be any arbitrary json config, not only plugins config.
Basically you need to assume that getConfig will return the entire config object of the player.

Reply:
done